docs/pages/_includes/services/security-transport-encryption-lead-in.md

```diff
@@ -0,0 +1,3 @@
+With transport encryption enabled, DC/OS {{ include.techName }} will automatically deploy all nodes with the correct configuration to encrypt communication via SSL. The nodes will communicate securely between themselves using SSL.{% if include.plaintext == "true" %} Optionally, plaintext communication can be left open to clients.{% endif %}
```

Comment:
SSL vs TLS?

Reply:
I went with SSL because that's what a lot of the services still call this in their docs /shrug